.github/updatePOMVersion.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+GIT_HASH=$(git rev-parse HEAD)
+if ! [ "$(git symbolic-ref --short HEAD)" = "beta" ]; then
+  echo "## Is MASTER or UPGRADE  branch, use pom.xml version"
+else
+  currentversion=$(sed -n '/<version>.*-SNAPSHOT<\/version>/ s/<version>\(.*\)-SNAPSHOT<\/version>/\1/p' pom.xml)
+  devversionname="$currentversion-${GIT_HASH:0:7}-SNAPSHOT"
+  echo "## Is DEVELOPER branch, update pom.xml version to: $devversionname"
+  mvn -B versions:set -DgenerateBackupPoms=false -DnewVersion="$devversionname"
+fi
```

Comment:
@anaiberta: this code is taken _as is_ from our internal build configuration but I'm not sure we should continue to use it. We no longer use the packages generated from PR builds so I think we could safely remove it. However, we could use this script to start using the master version + 100 in the beta branch. 

Could you also help me see whether I'm missing any special project configuration (eg. Mizuho, Corona, etc.) in this script?

Reply:
This was fixed in c98a704